core/core-funcs.el

```diff
@@ -141,48 +141,57 @@ Supported properties:
           (define-key (eval (car val)) (kbd (cdr val)) ',func)))))))
 
 (defun spacemacs/view-org-file (file &optional anchor-text expand-scope)
```

Comment:
I recently found this function and copied out the relevant bits to my personal files, because I wanted to make my Org buffers look as pretty and fancy as they do when spacemacs displays them.

I couldn't just call this function though, because there are many ways I end up in an org file.

Do you have any thoughts or considerations about extracting the parts in this that make an org buffer pretty, so that users could potentially use it (via org mode hooks, likely)?